### PR TITLE
Remove unused modules

### PR DESCRIPTION
**Pull request by AlexKurek**

### CODE REVIEW

File: crds/core/selectors.py [hunk 1]
```diff
@@ -92,8 +92,8 @@
 import numbers
 from collections import namedtuple
 import ast
-import copy
-from pprint import pprint as pp
+# import copy
+# from pprint import pprint as pp
```
Comment (zacharyburnett) [inline crds/core/selectors.py:96]:
```suggestion
```

File: crds/hst/acs_common.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 """Common imports and defs across all versions of ACS hooks.
 """
-import sys
-from collections import defaultdict
+# import sys
+# from collections import defaultdict
```
Comment (zacharyburnett) [inline crds/hst/acs_common.py:4]:
```suggestion
```

File: crds/jwst/locate.py [hunk 3]
```diff
@@ -8,15 +8,15 @@
 """
 import os.path
 import re
-import warnings
+# import warnings
```
Comment (zacharyburnett) [inline crds/jwst/locate.py:11]:
```suggestion
```

File: crds/jwst/locate.py [hunk 4]
```diff
@@ -8,15 +8,15 @@
...
 
-from asdf.tags.core import NDArrayType
-from stdatamodels.validate import ValidationWarning
+# from asdf.tags.core import NDArrayType
+# from stdatamodels.validate import ValidationWarning
```
Comment (zacharyburnett) [inline crds/jwst/locate.py:14]:
```suggestion
```

File: crds/jwst/locate.py [hunk 5]
```diff
@@ -8,15 +8,15 @@
...
 
 # =======================================================================
 
 from crds.core import rmap, config, utils, timestamp, log, exceptions
-from crds.certify import generic_tpn
+# from crds.certify import generic_tpn
```
Comment (zacharyburnett) [inline crds/jwst/locate.py:19]:
```suggestion
```

File: crds/jwst/locate.py [hunk 6]
```diff
@@ -25,20 +25,20 @@
...
 # from observatory-unique ways of specifying and caching Validator parameters.
 
-from crds.jwst import TYPES, INSTRUMENTS, FILEKINDS, EXTENSIONS, INSTRUMENT_FIXERS, TYPE_FIXERS
+from crds.jwst import TYPES, INSTRUMENTS, FILEKINDS #, EXTENSIONS, INSTRUMENT_FIXERS, TYPE_FIXERS
 
 from . import schema
 
 # definitively lists possible exp_types by instrument, or all instruments
-from .schema import get_exptypes
+# from .schema import get_exptypes
```
Comment (zacharyburnett) [inline crds/jwst/locate.py:33]:
```suggestion
```

File: crds/jwst/locate.py [hunk 7]
```diff
@@ -25,20 +25,20 @@
...
 
 get_row_keys_by_instrument = TYPES.get_row_keys_by_instrument
 get_item = TYPES.get_item
 suffix_to_filekind = TYPES.suffix_to_filekind
 filekind_to_suffix = TYPES.filekind_to_suffix
 get_all_tpninfos = TYPES.get_all_tpninfos
 
-from crds.jwst.pipeline import header_to_reftypes, header_to_pipelines
+# from crds.jwst.pipeline import header_to_reftypes, header_to_pipelines
```
Comment (zacharyburnett) [inline crds/jwst/locate.py:41]:
```suggestion
```

File: crds/roman/locate.py [hunk 8]
```diff
@@ -17,17 +17,17 @@
 """
 import os.path
 import re
-import warnings
+# import warnings
```
Comment (zacharyburnett) [inline crds/roman/locate.py:20]:
```suggestion
```

File: crds/roman/locate.py [hunk 9]
```diff
@@ -17,17 +17,17 @@
...
 from collections import namedtuple
 
-from asdf.tags.core import NDArrayType
+# from asdf.tags.core import NDArrayType
```
Comment (zacharyburnett) [inline crds/roman/locate.py:23]:
```suggestion
```

File: crds/roman/locate.py [hunk 10]
```diff
@@ -17,17 +17,17 @@
...
 
 # =======================================================================
 
 from crds.core import rmap, config, utils, timestamp, log, exceptions
-from crds.certify import generic_tpn
+# from crds.certify import generic_tpn
```
Comment (zacharyburnett) [inline crds/roman/locate.py:28]:
```suggestion
```

File: crds/roman/locate.py [hunk 11]
```diff
@@ -17,17 +17,17 @@
...
 from crds import data_file
-from crds.io import abstract
+# from crds.io import abstract
```
Comment (zacharyburnett) [inline crds/roman/locate.py:30]:
```suggestion
```

File: setup_data.py [hunk 12]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
-import sys
-import glob
+# import sys
+# import glob
```
Comment (zacharyburnett) [inline setup_data.py:3]:
```suggestion
```

File: test/bestrefs/test_table_effects.py [hunk 13]
```diff
@@ -1,9 +1,9 @@
 """This tests, through the use of bestrefs, the functioning of table effects."""
-from pathlib import Path
+# from pathlib import Path
```
Comment (zacharyburnett) [inline test/bestrefs/test_table_effects.py:2]:
```suggestion
```

File: test/bestrefs/test_table_effects.py [hunk 14]
```diff
@@ -1,9 +1,9 @@
...
 from pytest import mark
 
-from crds.core import log, utils
-from crds import data_file
+from crds.core import log  #, utils
+# from crds import data_file
```
Comment (zacharyburnett) [inline test/bestrefs/test_table_effects.py:6]:
```suggestion
from crds.core import log
```

File: test/core/test_rmap.py [hunk 15]
```diff
@@ -4,7 +4,7 @@
 import os
 import json
 import pickle
-import sys
+# import sys
```
Comment (zacharyburnett) [inline test/core/test_rmap.py:7]:
```suggestion
```

File: test/test_diff.py [hunk 16]
```diff
@@ -53,7 +53,7 @@
 ----------
 """
 import subprocess
-import os
+# import os
```
Comment (zacharyburnett) [inline test/test_diff.py:56]:
```suggestion
```

File: test/test_rowdiff.py [hunk 17]
```diff
@@ -3,7 +3,7 @@
 from pathlib import Path
 from pytest import mark
 
-from crds.core import log, utils
+from crds.core import log  #, utils
```
Comment (zacharyburnett) [inline test/test_rowdiff.py:6]:
```suggestion
from crds.core import log
```

**Comment by AlexKurek** [comment]
> I suggest removing the comments entirely to reduce clutter

I just removed it.

**Pull request closed without merging**